test/functional/page_objects/visualize_page.js

```diff
@@ -388,7 +388,9 @@ export function VisualizePageProvider({ getService, getPageObjects }) {
     }
 
     async addNewFilterAggregation() {
-      return await testSubjects.click('visEditorAddFilterButton');
+      await testSubjects.click('visEditorAddFilterButton');
+      // We should let new filter input to render completely before we enter any text into it.
+      return await PageObjects.common.sleep(500);
```

Comment:
__note:__ filed issue to investigate the underlying problem: https://github.com/elastic/kibana/issues/29833

Reply:
https://github.com/elastic/kibana/issues/29833

Reply:
Awesome, thanks!